docs/admin/predeployment_check.md

```diff
@@ -266,8 +266,8 @@ will be applied automatically on install).
 
 For non-RPM installations where `daos_server` has been built from source,
 `rmem_default` and `rmem_max` settings should be set to >= 1MB.
-Optionally, the `utils/rpms/sysctl.d/daos_server.conf` can be copied to
-`/etc/sysctl.d/` to apply the settings automatically on boot.
+Optionally, the `utils/rpms/daos_server.conf` can be copied to `/etc/sysctl.d/`
+to apply the settings automatically on boot.
```

Comment:
Should a suggestion of running `sysctl -p /etc/sysctl.d/10-daos_server.conf` so that these settings are applied even prior to a reboot?